aws/v11.4.1/README.md

```diff
@@ -0,0 +1,80 @@
+# :zap: Giant Swarm Release v11.4.1 for AWS :zap:
+
+This release re-activates the recent AWS release of high-availability (HA) masters while fixing issues with certain OIDC configurations.
```

Comment:
Can you add link to release notes with HA Masters details?

Reply:
Done.